app/helpers/harvests_helper.rb

```diff
@@ -19,20 +19,16 @@ def display_human_quantity(harvest)
       if harvest.unit == 'individual' # just the number
         number_to_human(harvest.quantity, strip_insignificant_zeros: true)
       elsif !harvest.unit.blank? # pluralize anything else
-        return pluralize(number_to_human(harvest.quantity, strip_insignificant_zeros: true), harvest.unit)
+        pluralize(number_to_human(harvest.quantity, strip_insignificant_zeros: true), harvest.unit)
```

Comment:
I'd be kind of inclined to refactor this method to return early, so theres less nested conditionals
```return unless !harvest.quantity.blank? && harvest.quantity > 0```